### ISSUE TITLE
[Bug] Handle failing to get guids

### ISSUE THREAD

**Issue by Black-Speedy** [luigi311/JellyPlex-Watched#159]
## Description
I have a request to make the script report when it gets an error for generating the watched lists for the different libraries. I just had an issue where it told me that it failed to generate the watched list for my jellyfin server, and I had no clue as to why. I then decided to try and print the names out when it tried to generate the list and I then found jellyfin hadn't recognized the files with the correct episodes. This made the script think that they were not there because they were reported as missing. This took a bit of time to figure out since the error I got was a python error which wasn't that helpful since it just said that guids = {"title": item["Name"]} was the issue under the get_guids function.

## TL;DR
I would like for it to report when something goes wrong and preferably what movie/show is the cause. This could also make it easier for the users to debug whether the script or jellyfin/plex is the issue.

**Comment by luigi311** [comment]
Interesting that Name was coming up missing, either way ive added handling for this so it will not output the information of the item if its missing information with debug level set to info. Give it a test in dev

**Comment by Black-Speedy** [comment]
It seems better from what I can tell
